_docs_integrate/request-and-response-introduction.md

```diff
@@ -411,8 +411,7 @@ In case you don't have a Relationship with the peer yet, opening the Relationshi
 If the peer decides to accept and to respond to your Request, again a LocalRequest and LocalResponse will be created at their side.
 However, the returned data differ.
 Instead of a ResponseWrapper inside a Message, a Relationship is returned which is in the status `pending` for now.
-It contains the RelationshipTemplate, as well as the changes the peer made to it, i.e. the created Request and Response.
-Only after you accept the RelationshipCreationChangeRequest (not the kind of Request discussed on this page), the LocalRequest with LocalResponse is created at your side and the peer will receive the information about the status change via a `consumption.incomingRequestStatusChanged` [event]({% link _docs_integrate/connector-events.md %}).
+It contains the RelationshipTemplate, as well as the Response to the Request.
```

Comment:
too much information is lost here, why did you remove the mention of the event?

Reply:
re-added it, idk why, this was a long time ago :)

Reply:
yeah, these docs PRs don't get enough love :(